src/zappa.coffee.md

```diff
@@ -852,19 +853,24 @@ Takes a function and runs it as a zappa app. Optionally accepts a port number, a
               switch k
                 when 'host' then host = v
                 when 'port' then port = v
+                when 'path' then ipc_path = v
                 else options[k] = v
 
       zapp = zappa.app(root_function,options)
       {server,app} = zapp
 
+      is_ipc = typeof ipc_path is 'string'
+
       server.on 'listening', ->
-        addr = server.address()
+        channel = if is_ipc then ipc_path else do (addr = server.address()) -> "#{addr.address}:#{addr.port}"
         debug """
-          Express server listening on #{addr.address}:#{addr.port} in #{app.settings.env} mode.
+          Express server listening on #{channel} in #{app.settings.env} mode.
           Zappa #{zappa.version} orchestrating the show.
 
         """
 
+      if is_ipc
```

Comment:
@shimaore Totally agree and I fixed that in a subsequent commit but I think you merged before I had a chance to force push (might submit as a separate PR).

BTW curious as to why you test for `ipc_path` and `host` differently (former using `?`). Am I missing something obvious here?

And appreciate for prompt review and merge!